src/index.ts

```diff
@@ -1363,12 +1365,50 @@ export interface CandidateCertificationRequirement {
     candidate?: Candidate;
     candidateCertification?: CandidateCertification;
     certification?: Certification;
+    certificationFileAttachments?: ToMany<CertificationFileAttachment>;
+    customDate1?: Date;
+    customDate10?: Date;
```

Comment:
Can this be sorted differently, so that customDate10 comes after customDate9? That would make it easier to read.